libsrc/apple2/sleep.s

```diff
@@ -0,0 +1,41 @@
+;
+; Colin Leroy-Mira <colin@colino.net>, 2024
+;
+; void __fastcall__ sleep(unsigned s)
+;
+;
+
+        .export         _sleep
+        .import         _apple2_slowdown
+        .import         _apple2_speedup
+        .importzp       tmp1
+
+MONWAIT         := $FCA8
```

Comment:
Are you aware that there's a pattern regarding ROM calls in the Apple II library? See i.e.:
https://github.com/cc65/cc65/blob/master/libsrc/apple2/cout.s
https://github.com/cc65/cc65/blob/master/libsrc/apple2/home.s
I'd appreciate if you would follow that pattern here. BTW: The ROM listing in the original Apple II reference manual calls the routing just `WAIT`. I propose to stick to that name.